---
bool success
string message
XBeeMessage[] observations
